clinica/iotools/bids_utils.py

```diff
@@ -496,6 +501,7 @@ def _write_bidsignore(bids_dir: Union[str, Path]) -> None:
 
 def write_modality_agnostic_files(
     study_name: str,
+    data_dict: dict,
```

Comment:
We should find a more explicit name than `data_dict`. I am usually not a big fan of puting `dict` or `list` suffixes to variable names because they are redundant with the type.

Instead, I'd propose something like `readme_data` or `readme_content`.

Reply:
From PyCharm: "Missing parameter data_dict in docstring"